File: Linguagem C/04_Lista_if_else/Ex_014.c
#include <stdio.h>

int main() {
    float h, ideal;
    char g;

    printf("Digite sua altura : ");
    scanf("%f", &h);
    printf("Digite seu sexo: ");
    scanf(" %c", &g);

    if ( g == 'm'){
        ideal = (72.7 * h) - 58;
        printf("%0.2f\n", ideal);
    } else if (g == 'f') {
        ideal = (62.1 * h) - 44.7;
         printf("%0.2f\n", ideal);
    }

    return 0;
}

File: Python/004_Lista_if_else/Ex_014.py
"""
14) Faça um programa que receba a altura (valor real) e o sexo (m ou f) de uma pessoa e calcule e mostre
seu peso ideal, utilizando as seguintes fórmulas (onde h corresponde à altura):
• Homens: (72,7 * h) – 58
• Mulheres: (62,1 * h) – 44,7
"""
altura = float(input("Digite a altura do usuario: \n"))
sexo = str(input("M ou F\n"))

if sexo == 'm':
    Peso = (72.7*altura)-58
    print("O peso ideal do usuario sera igual a: {:.2f}".format(Peso))
elif sexo == 'f':
    Peso = (62.1*altura)-44.7
    print("O peso ideal do usuario sera igual a: {:.2f}".format(Peso))
else: 
    print("Entrada invalida")
    